plot/chart.go

```diff
@@ -326,6 +328,56 @@ func (c *Chart) handleData(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (c *Chart) handleTradingHistoryData(w http.ResponseWriter, r *http.Request) {
+	pair := r.URL.Query().Get("pair")
+	if pair == "" {
+		w.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	w.Header().Set("Content-type", "text/csv")
+	w.Header().Set("Content-Disposition", "attachment;filename=history_"+pair+".csv")
+	w.Header().Set("Transfer-Encoding", "chunked")
+
+	var orders []model.Order
+	var err error
+	if c.paperWallet != nil {
+		orders, err = c.paperWallet.OrdersByPair(pair)
+		if err != nil {
+			log.Errorf("failed to create order by pair: %s", err.Error())
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+	}
+
+	csvFile, err := os.Create("history.csv")
+	if err != nil {
+		log.Errorf("failed creating file: %s", err.Error())
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	defer csvFile.Close()
+	csvWriter := csv.NewWriter(csvFile)
```

Comment:
Please, avoid to write files in disk. Instead of a file, you can use a buffer of bytes.
```suggestion
	buffer := bytes.NewBuffer(nil)
	csvWriter := csv.NewWriter(buffer)
```